Exercises/NoModules/Chapter 18/Soln18_03/DB.cpp

```diff
@@ -0,0 +1,113 @@
+// A mocked database
```

Comment:
check the comments I had in the other chapter.
Also for all the other files in this folder.